### DISCUSSION TITLE
Ask a small technical question?

### DISCUSSION THREAD

**Discussion by barnett-yuxiang** [mobile-dev-inc/Maestro#842] [General]
**Is Maestro based on appium? Or design it yourself？**

**Comment by igorsmotto** [comment]
Maestro is not based on appium

**Comment by erguerra** [comment]
Is there a way for testing Toast Messages in Android using Maestro? PS. assertVisible didn't work

**Reply by igorsmotto** [reply]
This feature was merged recently, target release is 1.26.0

**Reply by erguerra** [reply]
Nice is it already available at: "https://get.maestro.mobile.dev"
And also: Is there a new command for Toasts? Should I use `assertVisible` or extendedWaitUntil: visible: "Toast message" ?
Thanks for your answer

**Comment by axelniklasson** [comment]
Hi @erguerra! This has since been released so please try the latest version of Maestro and file a bug report [here](https://github.com/mobile-dev-inc/maestro/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BMaestro+version%2C+e.g.+v1.2.3%5D) if it doesn't work. Thanks!
